### PR TITLE
Update OpenXR documentation

### PR DESCRIPTION
**Pull request by MadeinPh1la**
* Add supported OpenXR extensions
* Updated minimum OpenXR version to 1.0.27

### CODE REVIEW

File: docs/guides/native/native-openxr-setup.md [hunk 1]
```diff
@@ -45,6 +45,11 @@ As of this writing the following extensions are supported. As always, the runtim
 - `XR_MND_headless`
 - `XR_ML_global_dimmer`
 - `XR_ML_frame_end_info`
+- `XR_KHR_convert_timespec_time`
+- `XR_KHR_vulkan_enable2`
```
Comment (aheaney) [inline docs/guides/native/native-openxr-setup.md:49]: nit- I can't actually comment above, but on line 42, it's  XR_KHR_vulkan_enable

File: docs/guides/native/native-openxr-setup.md [hunk 2]
```diff
@@ -45,6 +45,11 @@ As of this writing the following extensions are supported. As always, the runtim
...
+- `XR_MSFT_unbounded_reference_space`
+- `XR_ML_compat`
+- `XR_ML_ml2_controller_interaction`
```
Comment (aheaney) [inline docs/guides/native/native-openxr-setup.md:52]: I took a look at the output of xrEnumerateInstanceExtensionProperties in one of the ML OpenXR samples the other day. Looks like these are supported as well- (I imagine at least XR_ML_private isn't meant to be documented though)

XR_EXT_conformance_automation
XR_KHR_opengl_enable
XR_ML_private
XR_ML_session_performance_info
XR_EXT_debug_utils

File: docs/guides/native/native-openxr-setup.md [hunk 3]
```diff
@@ -45,6 +45,11 @@ As of this writing the following extensions are supported. As always, the runtim
 - `XR_MND_headless`
 - `XR_ML_global_dimmer`
 - `XR_ML_frame_end_info`
+- `XR_KHR_convert_timespec_time`
+- `XR_KHR_vulkan_enable2`
+- `XR_MSFT_unbounded_reference_space`
```
Comment (aheaney) [inline docs/guides/native/native-openxr-setup.md:50]: That's neat. How to openxr build apps on ML2 in large workspace might be a good topic for a guide or sample.

**Review by aheaney — APPROVED** [review]

File: docs/guides/native/native-openxr-setup.md (hunk 2 above)
Comment (MadeinPh1la) [reply docs/guides/native/native-openxr-setup.md:52]: Thank you, very much appreciated. I will check on this with our OpenXR group.

**Pull request merged**